pandas/core/api.py

```diff
@@ -25,6 +25,7 @@
                                   date_range, bdate_range)
 from pandas.tseries.tdi import TimedeltaIndex, Timedelta
 from pandas.tseries.period import Period, PeriodIndex
+from pandas.core.style import Styler
```

Comment:
If you're passing arguments to the Styler constructor, `precision, caption, uuid, table_styles`. especially `table_styles`. I can remove it if you want, though I do envision people using the Styler constructor as well as DataFrame.style.